integration-tests/integration/common/common.spec.js

```diff
@@ -373,6 +394,18 @@ And('I should see the success banner message {string}', $string => {
   cy.get('.moj-banner--success').contains($string).should('exist')
 })
 
+And('I should see a textarea with id {string}', $comment => {
+  cy.get(`#${$comment}`).should('exist')
+})
+
 And('I should see a count of {string}', $string => {
   cy.get('.govuk-heading-m').contains(`${$string}`).should('exist')
 })
+
+And('I should see the following summary list', $data => {
```

Comment:
We think this is unused?

Reply:
Sorry, this is needed. It is lift and shift from case-summary spec as this is needed in case comments spec too.